include/seqan3/std/charconv

```diff
@@ -0,0 +1,294 @@
+//!\ingroup charconv
+struct from_chars_result
+{
+    char const * ptr;  //!< A pointer to the input string.
+    errc ec;          //!< The error code.
+};
+
+//!\brief A BitmaskType used to specify floating-point formatting for std::to_chars and std::from_chars.
+//!\ingroup charconv
+enum class chars_format
+{
+    scientific = 0x1,             //!< Scientific notation, e.g. 3.991E-0003.
+    fixed = 0x2,                  //!< Fixed number of digits for precision.
+    hex = 0x4,                    //!< Hexadecimal notation. Ff specified in from_chars, prefix 0x,0X,x is not allowed.
+    general = fixed | scientific  //!< General use case.
+};
+
+} // namespace std
+
+#include <seqan3/std/charconv_detail.hpp> // structs definitions above are needed in this header
+
+namespace std
+{
+//!\cond
+// implementation detail taken from LLVM
+void to_chars(char*, char*, bool, int = 10) = delete;
+void from_chars(char const *, char const *, bool, int = 10) = delete;
+//!\endcond
+
+// -----------------------------------------------------------------------------
+// to_chars for integral types
+// -----------------------------------------------------------------------------
+
+/*!\brief Convert an integral into a char sequence.
+ * \ingroup charconv
+ * \tparam _Tp The type to convert to a char sequence; Must model std::Integral.
+ * \param[in]      first The start of the range to fill.
+ * \param[in]      last  The end of the range to fill.
+ * \param[in, out] value The value to store the parsed result in.
+ * \param[in]      base  The integer base of the format of the string to parse.
+ *                       Must be a value between 2 and 36 (inclusive).
+ * \returns A std::to_char_result. See detail section return value for more information.
+ *
+ * \details
+ *
+ * Converts value into a character string by successively filling the range
+ * [first, last), where [first, last) is required to be a valid range.
+ *
+ * value is converted to a string of digits in the given base (with no redundant
+ * leading zeroes). Digits in the range 10..35 (inclusive) are represented as
+ * lowercase characters a..z. If value is less than zero, the representation
+ * starts with a minus sign. The library provides overloads for all signed and
+ * unsigned integer types and for the type char as the type of the parameter value.
+ *
+ * ### Return value
+ * On success, returns a value of type to_chars_result such that ec equals
+ * value-initialized std::errc and ptr is the one-past-the-end pointer of the
+ * characters written. Note that the string is not NUL-terminated.
+ *
+ * On error, returns a value of type to_chars_result holding
+ * std::errc::value_too_large in ec, a copy of the value last in ptr, and leaves
+ * the contents of the range [first, last) in unspecified state.
+ *
+ * \sa https://en.cppreference.com/w/cpp/utility/to_chars
+ */
+template <typename char_type, std::Integral _Tp>
+    requires std::Same<char_type, char>
```

Comment:
Same thing here. If you do end up needing, please use the shorter concept syntax